addons/ai_apps/models/discuss_channel.py

```diff
@@ -0,0 +1,208 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import json
+import datetime
+import pytz
+from markupsafe import Markup
+from markdown import markdown
```

Comment:
Where's this lib coming from? Is it already in our dependencies?
If not, can it be added? Do we consider it safe to use?

Reply:
It was approved by odo